admin_app/src/app/dashboard/components/Performance.tsx

```diff
@@ -0,0 +1,147 @@
+import React from "react";
+import rows from "./rows";
+import Grid from "@mui/material/Unstable_Grid2";
+import Box from "@mui/material/Box";
+import SwapVertIcon from "@mui/icons-material/SwapVert";
+import TextField from "@mui/material/TextField";
+import FilterAltOutlinedIcon from "@mui/icons-material/FilterAltOutlined";
+import Pagination from "@mui/material/Pagination";
+
+const Performance: React.FC = () => {
+  return (
+    <>
+      <Box
```

Comment:
I'll split these out into components in future PRs. Just easy to have them in one place when building